### DISCUSSION TITLE
How to prepare dataset for the fine-tuning

### DISCUSSION THREAD

**Discussion by Saeedmatt3r** [openai/whisper#2136] [General]
I'm currently fine-tuning Whisper for my own language, guided by the blog post linked [here](https://huggingface.co/blog/fine-tune-whisper). I have a question regarding the injection of timestamps into the dataset. Specifically, I'm interested in how to insert timestamps in the middle of transcriptions during training. 

Additionally, I have a question that might be more appropriate for the original authors(@jongwook): How did you integrate timestamps? Did you use an automated method, such as a word aligner? How did you determine the correct placement of timestamps after specific words?

**Comment by gongouveia** [comment]
@Saeedmatt3r Hello, I am developing a tool to create synthetic datasets to fine tune whisper. [Whisper Temple](https://github.com/gongouveia/Whisper-Temple-Synthetic-ASR-Dataset-Generator) Allows you to record, upload audios, edit the transcriptions or generating artificial datasets. Furthermore you can export this transcriptions as a HF 🤗 dataset (this feature is almost being added).
Please follow my project if you thinks that this tools is useful for the open-source community and If you are interested in adding the timestamps feature to the dataset You can ope an issue in the project.

**Reply by Choi-YoungHyun** [reply]
I'm using windowsOS but your program is not working about theme...

**Reply by gongouveia** [reply]
@Choi-YoungHyun I am aware of that bug, does the program work other than that?
Please stick to the white theme for now. Follow the project to stay updated to when I fix this bug.
